jcl/src/java.base/share/classes/java/lang/ref/ReferenceQueue.java

```diff
@@ -191,17 +191,13 @@ public Reference<? extends T> remove(long timeout) throws IllegalArgumentExcepti
  *
  * @param		reference
  *					reference object to be enqueued.
- * @return		boolean
- *					true if reference is enqueued.
- *					false if reference failed to enqueue.
  */
-boolean enqueue(Reference<? extends T> reference) {
+void enqueue(Reference<? extends T> reference) {
 	/*[PR CMVC 96472] deadlock loading sun.misc.Cleaner */
 	/*[PR 102259] call Cleaner.clean(), do not enqueue */
 	if (reference instanceof Cleaner) {
 		reference.dequeue();
 		((Cleaner)reference).clean();
-		return true;
```

Comment:
The `return;` statements must stay (here and what was line 213).
Also `enqueue0()` must change.